api/server/server_windows.go

```diff
@@ -0,0 +1,19 @@
+// +build windows
+
+package server
+
+import (
+	"fmt"
+	"github.com/docker/docker/engine"
```

Comment:
non-stdlib imports are be separated with an empty line (my observation in repo in general)